src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -30,8 +32,10 @@
 
 /**
  * The ConsumerConfiguration class specifies the configuration for creating a JetStream consumer on the client and
- * if necessary the server.
+ * if necessary the server. 
```

Comment:
Can you set your editor to trim trailing spaces?

Reply:
Done